### DISCUSSION TITLE
SEQR filter results

### DISCUSSION THREAD

**Discussion by Tulasi08** [broadinstitute/seqr#4231] [General]
Hi SEQR Team,

We had 2 general doubts and wanted to clarify it with you. 
Question 1 - Even though when we used the below-mentioned filters for 2 projects that are having similar set of samples (numbers), in one we got variants and in another it was not giving variants with same set. Please note location is empty for both. If we now give a location (i.e. a gene list), very few variants (<20) are coming up. Question is to re-confirm if Seqr is doing any calculation at back end, or is it related to variant distribution across samples across projects?
<img width="711" alt="image" src="https://github.com/user-attachments/assets/e57ce987-24ba-41f0-8024-8fd197675d40">
<img width="736" alt="image" src="https://github.com/user-attachments/assets/7f2b03cb-9d5a-4c0a-a4eb-d7d8cf2f6010">
<img width="937" alt="image" src="https://github.com/user-attachments/assets/e93b9069-63ba-4b3b-a079-964e785213a6">

Question 2- On a general note, why Seqr does not allow downloading >1000 variants? Is there any restriction. 

Thanks and Regards,
Tulasi S

**Comment by hanars** [comment]
So the issue, according to the error message in your screenshot, is that you have too many variants coming back in compound het search and you need to apply stricter filters. The reason you are seeing that issue in some projects and not others probably has to do with your family structures - full trios and quads return far fewer compound het variants than duos and singletons. Alternatively, perhaps one project has noisier data so has fewer variants passing quality filters than the other. 
> Question is to re-confirm if Seqr is doing any calculation at back end, or is it related to variant distribution across samples across projects?

I do not understand the question, but what happens in this case is that  seqr does a search, sees that too many variants will come back, and shows an error instead of crashing the entire app by trying to execute a query that is too big

Regarding Question 2, this is because this is a computationally intensive process that is impossible to reliably support, and moreover exporting thousands of variants is a misuse of seqr. If you want to get a subset of a few thousand variants that meet some criteria, you should be using computational tools to process your raw VCF and do that as a computational experiment run by someone who is able to delve into raw data. seqr is a user-focused application for the discovery of rare causal variants in rare disease families - there is no use case where a human being should be looking at thousands of variants to try to solve a case. Like all software systems, we have optimized seqr's databases to be able to run the kinds of queries we need to support for our application, and we are not able to support other kinds of queries.
